### PR TITLE
Add greeting for contributors on raising their first PR or issue

### PR DESCRIPTION
**Pull request by winter-code**
Messages ( for issues and PRs) can be customized. This is recommended since a hello message on successfully raising a PR will motivate new contributors. :rocket:

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Here's a :candy: while we look into your'' first issue'
```
Comment (tomdyson) [inline .github/workflows/greetings.yml:12]: I think the apostrophe in `Here's` needs to be escaped with a second single quote, and there shouldn't be two single quotes after `your`. Colons also have tricky escape sequences:

https://yaml.org/spec/1.2/spec.html#id2788097

Perhaps we should go for a more straightforward

> 'Thanks for submitting your first issue to Wagtail!'

and 

> 'Thanks for submitting your first PR to Wagtail!'

**Review by tomdyson — CHANGES_REQUESTED** [review]
Thanks @winter-code! This will be a nice welcome to new contributors. I've suggested a simpler message, partly to help with tricky yaml escaping.

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (winter-code) [reply .github/workflows/greetings.yml:12]: Looks good! This is my first time working with yaml files and I did try out these messages on my own repos but wasn't sure what was wrong with the message that the :candy: didn't appear (which was sad)

Making the changes as suggested by you! :)

**Comment by lb-** [comment]
@vossisboss - when you get a min - do you mind reviewing this in the context of community. 

I think it would be great to have this in but it's been a while since this was created so maybe cast your eyes over it.

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thanks for submitting your first issue to Wagtail!'
+        pr-message: 'Thanks for submitting your first PR to Wagtail!'
```
Comment (vossisboss) [inline .github/workflows/greetings.yml:13]: I think this is a great idea! I'd just like to suggest we give the PR submitters some reassurance that we will review their work as soon as we can.

```suggestion
        pr-message: 'Thanks for submitting your first PR to Wagtail! We know how hard you worked on this. Please give us some time to review, and we'll be in touch as soon as we can.'
```

**Review by vossisboss — CHANGES_REQUESTED** [review]
Requesting one small change. Otherwise I think this is a great contribution!

**Comment by winter-code** [comment]
I will make the necessary changes! :D

**Comment by thibaudcolas** [comment]
Thank you for giving this a go @winter-code! We’ve just discussed this at our weekly core contributor team meetings, and have decided to not go further with this. We think it’s better for us to personalise our greetings / thanks to contributors, particularly new contributors.

There is a chance we might not do so consistently but we think this is worth the tradeoff, since personal greetings by other contributors will be more meaningful.

**Pull request closed without merging**